pipeline/transformer/transformer_mets/src/test/scala/weco/pipeline/transformer/mets/transformers/PremisAccessConditionsTest.scala

```diff
@@ -50,19 +52,40 @@ class PremisAccessConditionsTest
     }
 
     describe("extracting values from a rightsMD section") {
-      it("pulls out the copyrightNote for the licence") {
+      it("pulls out the copyrightNote where rightsBasis is 'Copyright'") {
         PremisAccessConditions(
           openInCopyrightRightsMD
         ).copyrightNote shouldBe Some(
           "In copyright"
         )
       }
+
+      it("pulls out the licenceNote where rightsBasis is 'License'") {
+        PremisAccessConditions(
+          openCCBYNCRightsMD
+        ).copyrightNote shouldBe Some("CC-BY-NC")
+      }
+
+      it("pulls out the copyrightNote where rightsBasis is not specified") {
+        PremisAccessConditions(
+          openMixedCCBYNCWithCopyrightRightsMD
+        ).copyrightNote shouldBe Some(
+          "In copyright"
+        )
+      }
+
       it("pulls out the rightsGrantedNote for the access status") {
         PremisAccessConditions(
           openInCopyrightRightsMD
         ).useRightsGrantedNote shouldBe Some("Open")
       }
 
+      it("pulls out the copy for the access status") {
+        PremisAccessConditions(
+          openInCopyrightRightsMD
+        ).useRightsGrantedNote shouldBe Some("Open")
+      }
+
```

Comment:
This is identical to the test above. I assume `openInCopyrightRightsMD` should be replaced with `openCCBYNCRightsMD`?

Reply:
I think this test might be left in accidentally after a copy / paste. I'll remove it.